Makefile

```diff
@@ -293,8 +298,14 @@ clean-easyjson-srcs:
 	@echo "+ $@"
 	$(SILENT)find . -name '*_easyjson.go' -exec rm {} \;
 
+COMPLIANCEOPERATOR_FILES = $(shell find pkg/complianceoperator/ -type f -name '*.go')
+pkg/complianceoperator/api/v1alpha1/zz_generated.deepcopy.go: $(CONTROLLER_GEN_BIN) $(COMPLIANCEOPERATOR_FILES) ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+	$(CONTROLLER_GEN_BIN) object:headerFile="operator/hack/boilerplate.go.txt" paths="./pkg/complianceoperator/..."
```

Comment:
I'd prefer to have a separate copy of the boilerplate for complianceoperator, but I don't feel strongly about this.

Reply:
Done, I created empty file like was before